static/src/js/components/GranuleImage/__tests__/GranuleImage.test.js

```diff
@@ -1,70 +1,68 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
+import {
+  render,
+  screen
+} from '@testing-library/react'
+import { act } from 'react-dom/test-utils'
+import userEvent from '@testing-library/user-event'
+import '@testing-library/jest-dom'
 import GranuleImage from '../GranuleImage'
 
-Enzyme.configure({ adapter: new Adapter() })
-
-function setup(overrideProps) {
-  const props = {
-    imageSrc: '',
-    ...overrideProps
-  }
-
-  const enzymeWrapper = shallow(<GranuleImage {...props} />)
-
-  return {
-    enzymeWrapper,
-    props
-  }
+const setup = (props) => {
+  act(() => {
```

Comment:
Is this `act` necessary?

Reply:
No I think I was using it earlier and didn't remove it!